changes/2020-07-14_multi-keyring-require-generation/change.md

```diff
@@ -0,0 +1,108 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Require Generation in the Multi-Keyring
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                                                                           |
+| ------------------------------------------------------------------------------------------------- |
+| [Multi-Keyring](../../framework/multi-keyring.md)                                                 |
+| [Multi keyring generator](https://github.com/awslabs/aws-encryption-sdk-specification/issues/114) |
```

Comment:
np: we should stick with either `multi-keyring` or `multi keyring`

Reply:
Agreed, the second was copied from the issue title. Will fix.